Kubernetes/legos/k8s_execute_helm_command/k8s_execute_helm_command.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2024 unSkript.com
+# All rights reserved.
+#
+import subprocess
+from pydantic import BaseModel, Field
+
+class InputSchema(BaseModel):
+    helm_command: str = Field(
+        title='Helm Command',
+        description='Helm command to execute in the K8s Cluster'
+    )
+
+def k8s_execute_helm_command_printer(data: str):
+    if not data:
+        return 
+    
+    print(data)
+
+
+def k8s_execute_helm_command(handle, helm_command: str) -> str:
+    """k8s_execute_helm_command executes the given helm command in the k8s cluster
+
+       :type handle: object
+       :param handle: Object returned from the Task validate method
+
+       :type helm_command: str
+       :param helm_command: Helm Command that need to be executed 
+
+       :rtype: String, Output of the given helm command. Empty string in case of error
+    """
+    retval = None 
+    if handle.client_side_validation is not True:
+        print(f"K8S Connector is invalid: {handle}")
+        return str()
+    
+    if not helm_command:
+        print(f"Given helm command is empty, cannot proceed further!")
+        return str()
+    
+    config_file = None
+    try:
+        config_file = handle.temp_config_file 
```

Comment:
why are we doing this way? Why not define a helm lambda, just like we have for running kubectl command and call that??

Reply:
As we discussed, will keep the logic of running subprocess in the lego itself.